compose.yaml

```
@@ -1,47 +1,35 @@
 ﻿services:
-  structurizr:
-    image: structurizr/lite
+  postgres-task:
+    image: postgres:latest
+    environment:
+      - POSTGRES_PASSWORD=postgres
+      - POSTGRES_USER=postgres
+      - POSTGRES_DB=todoapp
     ports:
-      - "8080:8080"
+      - "5433:5432" # Different host port to avoid conflict
     volumes:
-      - ./docs:/usr/local/structurizr
-  
-  task-ms:
-    image: task-ms
-    build:
-      context: .
-      dockerfile: TaskMicroservice/Dockerfile
-
+      - postgres-task-data:/var/lib/postgresql/data
+    healthcheck:
+      test: [ "CMD-SHELL", "pg_isready -U postgres" ]
+      interval: 10s
+      timeout: 5s
+      retries: 5
 
-  usermicroservice:
-    image: usermicroservice
-    build:
-      context: .
-      dockerfile: UserMicroservice/Dockerfile
-    depends_on:
-      - postgres
-      - unleash
-    environment:
-      - ConnectionStrings__DefaultConnection=Host=postgres;Database=userdb;Username=root;Password=password
-      - Unleash__ApiUrl=http://unleash:4242/api/
-      - Unleash__ApiToken=default:development.unleash-insecure-api-token
-  
-  postgres:
+  postgres-user:
     image: postgres:latest
     environment:
-      POSTGRES_PASSWORD: password
-      POSTGRES_USER: root
+      - POSTGRES_PASSWORD=password
+      - POSTGRES_USER=ToDoOrNotToDo
+      - POSTGRES_DB=ToDoOrNotToDo
     ports:
-      - "5432:5432"
+      - "5434:5432"
```

```suggestion
      - "5432:5432"
```
We are expecting port the user database to be available on port 5432 in the appsettings, why are we exposing a different port? As far as I can see, there is nothing else on 5432, since the task database is using 5433.